aten/src/ATen/native/native_functions.yaml

```diff
@@ -271,6 +271,9 @@
 - func: ones_like(Tensor self, *, Type dtype) -> Tensor
   variants: function
 
+- func: pairwise_distance(Tensor x1, Tensor x2, double p, double eps) -> Tensor
```

Comment:
You can (and probably should, to match the python API) set a default value for eps and p.